tests/forte/data/data_store_serialization_test.py

```diff
@@ -36,19 +39,29 @@ def setUp(self) -> None:
         DataStore._type_attributes = {
             "ft.onto.base_ontology.Document": {
                 "attributes": {
-                    "sentiment": 4,
-                    "classifications": 5,
+                    "document_class": {"index": 4, "type": (list, (str,))},
```

Comment:
We are adding a new attribute. So will there be an error if we don't include `document_class` here?